internal/provider/resource_group_members.go

```diff
@@ -0,0 +1,378 @@
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  deliverySettingsDefault,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice([]string{"ALL_MAIL", "DAILY", "DIGEST",
+					"DISABLED", "NONE"}, false)),
+			},
+			"status": {
+				Description: "Status of member.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"id": {
+				Description: "The unique ID of the group member. A member id can be used as a member request URI's memberKey.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+		},
+	}
+}
+
+func resourceGroupMembersCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	// use the meta value to retrieve your client from the provider configure method
+	client := meta.(*apiClient)
+	groupId := d.Get("group_id").(string)
+
+	log.Printf("[DEBUG] Creating Group Members in group %s", groupId)
+
+	directoryService, diags := client.NewDirectoryService()
+	if diags.HasError() {
+		return diags
+	}
+
+	membersService, diags := GetMembersService(directoryService)
+	if diags.HasError() {
+		return diags
+	}
+
+	members := d.Get("members").(*schema.Set)
+	for _, mMap := range members.List() {
+		memb := mMap.(map[string]interface{})
+
+		memberObj := directory.Member{
+			Email:            memb["email"].(string),
+			Role:             memb["role"].(string),
+			Type:             memb["type"].(string),
+			DeliverySettings: memb["delivery_settings"].(string),
+		}
+
+		log.Printf("[DEBUG] Creating Group Member %q in group %s: %#v", memberObj.Email, groupId, memberObj.Email)
+
+		_, err := membersService.Insert(groupId, &memberObj).Do()
+		if err != nil {
+			return diag.FromErr(err)
+		}
+	}
+
+	d.SetId(fmt.Sprintf("groups/%s", groupId))
+
+	return resourceGroupMembersRead(ctx, d, meta)
+}
+
+func resourceGroupMembersRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	// use the meta value to retrieve your client from the provider configure method
+	client := meta.(*apiClient)
+
+	directoryService, diags := client.NewDirectoryService()
+	if diags.HasError() {
+		return diags
+	}
+
+	membersService, diags := GetMembersService(directoryService)
+	if diags.HasError() {
+		return diags
+	}
+
+	groupId := d.Get("group_id").(string)
+
+	membersObj, err := membersService.List(groupId).Do()
+	if err != nil {
+		return handleNotFoundError(err, d, d.Id())
+	}
+
+	configMembers := d.Get("members").(*schema.Set)
+
+	members := make([]interface{}, len(membersObj.Members))
+	for i, member := range membersObj.Members {
+
+		// Use value if present or default as "delivery_settings" is not provided by API
+		deliverySettings := deliverySettingsDefault
+
+		for _, cm := range configMembers.List() {
+			cMem := cm.(map[string]interface{})
+			if cMem["email"].(string) == member.Email {
+				deliverySettings = cMem["delivery_settings"].(string)
```

Comment:
since we can't distinguish unset vs empty in the old SDK. I think you want to check if deliverySettings is empty and if it is then set it to default